tests/admin/test-class-boldgrid-backup-admin-db-import.php

```diff
@@ -0,0 +1,205 @@
+<?php //phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
+/**
+ * File: test-class-boldgrid-backup-admin-db-import.php
+ *
+ * @link  https://www.boldgrid.com
+ * @since SINCEVERSION
+ *
+ * @package    Boldgrid_Backup
+ * @subpackage Boldgrid_Backup/tests/admin
+ * @copyright  BoldGrid
+ * @version    SINCEVERSION
+ * @author     BoldGrid <support@boldgrid.com>
+ */
+
+/**
+ * Class: Test_Boldgrid_Backup_Admin_Db_Import
+ *
+ * @since xxx
+ */
+
+class Test_Boldgrid_Backup_Admin_Db_Import extends \WP_UnitTestCase {
+
+	/**
+	 * Setup.
+	 *
+	 * @since xxx
+	 */
+	public function setUp() {
+		global $wpdb;
+
+		$this->core = new \Boldgrid_Backup_Admin_Core();
+
+		$this->test_lines = [
+			'1234',
+			'abcd',
+			'6789',
+			'efgh',
+		];
+
+		$this->original_view_string = '/*!50001 DROP VIEW IF EXISTS `test_view`*/;\n/*!50001 CREATE ALGORITHM=UNDEFINED */\r/*!50013 DEFINER=`original_user`@`localhost` SQL SECURITY DEFINER */\n/*!50001 VIEW `test_view` AS select `wp_options`.`option_id` AS `option_id`,`wp_options`.`option_name` AS `option_name`,`wp_options`.`option_value` AS `option_value`,`wp_options`.`autoload` AS `autoload` from `wp_options` */;';
+		$this->original_view_lines  = [
+			'/*!50001 DROP VIEW IF EXISTS `test_view`*/;',
+			'/*!50001 CREATE ALGORITHM=UNDEFINED */',
+			'/*!50013 DEFINER=`original_user`@`localhost` SQL SECURITY DEFINER */',
+			'/*!50001 VIEW `test_view` AS select `wp_options`.`option_id` AS `option_id`,`wp_options`.`option_name` AS `option_name`,`wp_options`.`option_value` AS `option_value`,`wp_options`.`autoload` AS `autoload` from `wp_options` */;',
+		];
+
+		$this->expected_view_lines = [
+			'/*!50001 DROP VIEW IF EXISTS `test_view`*/;',
+			'/*!50001 CREATE ALGORITHM=UNDEFINED */',
+			'/*!50013 DEFINER=`' . DB_USER . '`@`localhost` SQL SECURITY DEFINER */',
+			'/*!50001 VIEW `test_view` AS select `wp_options`.`option_id` AS `option_id`,`wp_options`.`option_name` AS `option_name`,`wp_options`.`option_value` AS `option_value`,`wp_options`.`autoload` AS `autoload` from `wp_options` */;',
+		];
+	}
+
+	public function test_instance() {
```

Comment:
This is not needed.  The `$core` class property in `Boldgrid_Backup_Admin_Db_Import` should remain private.  If really needed, then create a public getter method in the class.